app/controllers/graphql_controller.rb

```diff
@@ -15,6 +14,21 @@ def execute
   end
 
   private
+  def current_user
+    credential = setup_credentials
+
+    return unless credential
```

Comment:
just found a bug in my code you copied from :(
```suggestion
    return unless credential&.ok?
```

Reply:
I put the ok? test inside the credential method. So current_user_id gives back the id, if ok? passes

Reply:
That's ok - though we then double check the credential being present and then ok. Up to you whether to move it to one check location here.